pkg/typeutil/conversion_test.go

```diff
@@ -35,3 +38,23 @@ func (s *testUint64BytesSuite) TestUint64ToBytes(c *C) {
 	str := "\x00\x00\x00\x00\x00\x00\x03\xe8"
 	c.Assert(b, DeepEquals, []byte(str))
 }
+
+var _ = Suite(&testJSONSuite{})
+
+type testJSONSuite struct{}
+
+func (s *testJSONSuite) TestJSONToUint64Slice(c *C) {
+	type testArray struct {
+		Array []uint64 `json:"array"`
+	}
+	a := testArray{
+		Array: []uint64{1, 2, 3},
```

Comment:
What about adding an invalid test case?

Reply:
@rleungx I mean a case that the `JSONToUint64Slice` function which will return nil and false.